tools/generate_input_variation_test_from_models.py

```diff
@@ -0,0 +1,101 @@
+import os
+from pathlib import Path
+from tools.collect_metrics import load_pickle, InputVarPerOp
+
+
+class InputVarTestExporter(InputVarPerOp):
+    def export_tests(self, basedir: Path):
+        # undo _join_br
+        def _unjoin_br(str_br: str):
+            return str_br.split(",<br>")
+
+        sort_by_opname = dict(sorted(self.items()))
+        basedir.mkdir(parents=True, exist_ok=True)
+        for opname, inputs_variations in sort_by_opname.items():
+            inputs_strings = [_unjoin_br(input_variations) for input_variations in inputs_variations.keys()]
+            opname_ = opname.replace(".", "_")
+            filename = f"test_{opname_}.py"
+            text = f"""
```

Comment:
can we move this to a separate file, smth like `aten_test_template`?

Reply:
Had moved to `tools/aten_test.tmpl`